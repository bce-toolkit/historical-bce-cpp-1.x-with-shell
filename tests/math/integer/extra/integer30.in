68024595121315284412431371454861433745289593954206986892748121847656447541775810099412340507329929980367694350046507263308620881950075111229234932174546783773966473299089538154662
23723761220985910255923199331520886242832587364510271266778954463219912322185134901527762438923208
