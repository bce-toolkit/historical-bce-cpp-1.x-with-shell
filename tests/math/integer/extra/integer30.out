68024595121315284412431371454861433745289593954206986892748121847656447541775810123136101728315840236290893681567393506141208246460346378008189395394459105959101374826851977077870
68024595121315284412431371454861433745289593954206986892748121847656447541775810075688579286344019724444495018525621020476033517439803844450280468954634461588831571771327099231454
1613799251812326885722037824765691611143744025033073516801381443612844515327212473436000377528641521534836844706440880380463519526330174027408744520200925463014130365679423501170294140602236403119877646533311240025592833921267283918413194802157774614548509776766292526645195696
2867361312890853800568427275288754128656749812851723029256108353806822430355818980
4053207316726438998058533032920137689969095149118859215265434124711842814158784338643681369266822
0
1
0
0
1
1
